engine/execution/ingestion/engine.go

```diff
@@ -1101,6 +1101,13 @@ func (e *Engine) saveExecutionResults(
 	if err != nil {
 		return nil, fmt.Errorf("cannot build chunk data pack: %w", err)
 	}
+	for _, event := range executionResult.ServiceEvents {
+		e.log.Info().
+			Uint64("block_height", result.ExecutableBlock.Height()).
+			Hex("block_id", logging.Entity(result.ExecutableBlock)).
+			Str("event_type", event.Type).
+			Msg("service event emitted")
+	}
```

Comment:
do we need this ?

Reply:
It's quite useful for debugging to see that service events are detected by the EN. They're very infrequent (2 events per epoch) if you're worried about spamming the logs.